docs/resources/aviatrix_dcf_mwp_policy_list.md

```diff
@@ -0,0 +1,135 @@
+resource "aviatrix_dcf_mwp_policy_list" "test" {
+  name = "Test policy"
+  policies {
+    name             = "df-policy-1"
+    action           = "DENY"
+    priority         = 1
+    protocol         = "ICMP"
+    logging          = false
+    watch            = false
+    src_smart_groups = [
+      "f15c9890-c8c4-4c1a-a2b5-ef0ab34d2e30"
+    ]
+    dst_smart_groups = [
+      "82e50c85-82bf-4b3b-b9da-aaed34a3aa53"
+    ]
+    tls_profile = "def000ad-6000-0000-0000-000000000001"
+  }
+
+  policies {
+    name             = "df-policy"
+    action           = "PERMIT"
+    priority         = 0
+    protocol         = "TCP"
+    src_smart_groups = [
+      "7e7d1573-7a7a-4a53-bcb5-1ad5041961e0"
+    ]
+    dst_smart_groups = [
+      "f05b0ad7-d2d7-4d16-b2f6-48492319414c"
+    ]
+
+    port_ranges {
+      hi = 50000
+      lo = 49000
+    }
+  }
+}
+```
+```hcl
+# Create an Aviatrix Distributed Firewalling MultiWriter (MWP) Policy List
+resource "aviatrix_distributed_firewalling_policy_list" "test" {
+  name = "Test policy"
+
+  policies {
+    name             = "df-policy"
+    action           = "PERMIT"
+    priority         = 0
+    protocol         = "TCP"
+    src_smart_groups = [
+      "7e7d1573-7a7a-4a53-bcb5-1ad5041961e0"
+    ]
+    dst_smart_groups = [
+      "f05b0ad7-d2d7-4d16-b2f6-48492319414c"
+    ]
+
+    port_ranges {
+      hi = 50000
+      lo = 49000
+    }
+  }
+
+  policies {
+    name                     = "df-policy-1"
+    action                   = "DEEP_PACKET_INSPECTION_PERMIT"
+    priority                 = 1
+    protocol                 = "ICMP"
+    logging                  = false
+    watch                    = false
+    exclude_sg_orchestration = true
+    src_smart_groups         = [
+      "f15c9890-c8c4-4c1a-a2b5-ef0ab34d2e30"
+    ]
+    dst_smart_groups         = [
+      "82e50c85-82bf-4b3b-b9da-aaed34a3aa53"
+    ]
+    web_groups               = [
+      "6bff3e91-3707-4582-9ea6-70e37b08760b"
+    ]
+    flow_app_requirement     = "TLS_REQUIRED"
+    decrypt_policy           = "DECRYPT_ALLOWED"
+    port_ranges {
+      hi = 50000
+      lo = 49000
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+### Required
+* `name` - (Required) Name of the policy list
+* `policies` - (Required) List of policies.
+    * `name` - (Required) Name of the policy.
+    * `action` - (Required) Action for the policy. Must be one of PERMIT, DENY, DEEP_PACKET_INSPECTION_PERMIT or INTRUSION_DETECTION_PERMIT.
+    * `priority` - (Optional)  Priority for the policy. Default: 0. Type: Integer.
+    * `protocol` - (Required) Protocol for the policy. Must be one of TCP, UDP, ICMP or ANY.
+    * `src_smart_groups` - (Required) List of Smart Group UUIDs for the source for the policy.
+    * `dst_smart_groups` - (Required) List of Smart Group UUIDs for the destination for the policy.
+    * `web_groups` - (Optional) List of Web Group UUIDs for the policy.
```

Comment:
These are sets technically not lists although I don't think that distinction matters too much.